.circleci/config.yml

```diff
@@ -95,7 +95,7 @@ jobs:
             - target
             - testsuite
 
-  test-firefox:
```

Comment:
Let's keep the changes to this file.

Reply:
https://github.com/hoodmane/libffi-emscripten/pull/13/commits/5254d393f55b8bef83aa9172984352eeb727277f